docs/ref/forms/api.txt

```diff
@@ -830,107 +830,228 @@ The field-specific output honors the form object's ``auto_id`` setting::
 
-    >>> initial = {'subject': 'welcome'}
-    >>> unbound_form = ContactForm(initial=initial)
-    >>> bound_form = ContactForm(data, initial=initial)
-    >>> print(unbound_form['subject'].value())
-    welcome
-    >>> print(bound_form['subject'].value())
-    hi
+    :attr:`~django.forms.BoundField.html_name` with an ``initial-`` prefix::
+
+        >>> f = ContactForm()
+        >>> print(f['subject'].html_initial_name)
+        initial-subject
+
+.. attribute:: BoundField.html_name
+
+    The name of the field that will be used in the input element's name
+    field. This takes the form prefix into account, if it has been set.
 
 .. attribute:: BoundField.id_for_label
 
-Use this property to render the ID of this field. For example, if you are
-manually constructing a ``<label>`` in your template (despite the fact that
-:meth:`~BoundField.label_tag` will do this for you):
+    Use this property to render the ID of this field. For example, if you are
+    manually constructing a ``<label>`` in your template (despite the fact that
+    :meth:`~BoundField.label_tag` will do this for you):
+
+    .. code-block:: html+django
+
+        <label for="{{ form.my_field.id_for_label }}">...</label>{{ my_field }}
+
+    By default, this will be the field's name prefixed by ``id_``
+    ("``id_my_field``" for the example above). You may modify the ID by setting
+    :attr:`~django.forms.Widget.attrs` on the field's widget. For example,
+    declaring a field like this::
+
+        my_field = forms.CharField(widget=forms.TextInput(attrs={'id': 'myFIELD'}))
+
+    and using the template above, would render something like:
+
+    .. code-block:: html
+
+        <label for="myFIELD">...</label><input id="myFIELD" type="text" name="my_field" />
+
+.. attribute:: BoundField.is_hidden
+
+    Returns ``True`` if this :class:`~django.forms.BoundField`'s widget is
+    hidden.
+
+.. attribute:: BoundField.label
+
+    The :attr:`~django.forms.Field.label` of the field. This is used in
+    :meth:`~BoundField.label_tag`.
+
+.. attribute:: BoundField.name
 
-.. code-block:: html+django
+    The name of this field in the form::
 
-    <label for="{{ form.my_field.id_for_label }}">...</label>{{ my_field }}
+        >>> f = ContactForm()
+        >>> print(f['subject'].name)
+        subject
+        >>> print(f['message'].name)
+        message
 
-By default, this will be the field's name prefixed by ``id_``
-("``id_my_field``" for the example above). You may modify the ID by setting
-:attr:`~django.forms.Widget.attrs` on the field's widget. For example,
-declaring a field like this::
+Methods of ``BoundField``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. method:: BoundField.__iter__()
+
+    Yields rendered strings that comprise all widgets in this BoundField.
+
+    This really is only useful for RadioSelect widgets, so that you can
+    iterate over individual radio buttons in a template.
+
+.. method:: BoundField.as_hidden(attrs=None, \*\*kwargs)
```

Comment:
I definitely think using a Widget there is better. But a quick github code search shows that at least `at_hidden` is used frequently.